scripts/zones/Bastok_Markets/Zone.lua

```diff
@@ -17,24 +17,22 @@ end
 function onZoneIn(player,prevZone)
     local cs = -1
 
-    -- SOA 1-1 Optional CS
-    if 
-        ENABLE_SOA and 
-        player:getCurrentMission(SOA) == dsp.mission.id.soa.RUMORS_FROM_THE_WEST and 
-        player:getCharVar("SOA_1_CS2") == 0 
-    then
-        cs = 22
-    end
-
     -- FIRST LOGIN (START CS)
-    if (player:getPlaytime(false) == 0) then
-        if (OPENING_CUTSCENE_ENABLE == 1) then
-            --cs = 0
-        end
+    if player:getPlaytime(false) == 0 and OPENING_CUTSCENE_ENABLE == 1 then
```

Comment:
don't we still want to set pos and homepoint even if opening cs is disabled? by merging those 2 if's that won't happen anymore.